fuzz/client.c

```diff
@@ -25,6 +26,23 @@ extern int rand_predictable;
 /* unused, to avoid warning. */
 static int idx;
 
+#define FUZZTIME 1485898104
+
+#ifdef _WIN32
+# ifdef _TIME64_T_DEFINED
+time64_t _time64(time64_t *t)
+# else
+time_t _time(time_t *t)
```

Comment:
To clarify and summarize. Goal is to provide replacement implementation for subroutine that returns time, so that linker is tricked to use it instead of libc/crt, right? Target lies "elsewhere", i.e. not in this compilation unit. On Windows there are thee possible options, and which of the three is used depends on compiler, its version and bit-ness, and possibly even flags. One can dive into all this parameters and cook some #ifdef spaghetti. *Or* one can simply provide *several* implementations with much simpler #idef logic and let *something* happen. Normally following would do:
```
#if defined(_WIN32) && defined(_TIME64_T_DEFINED)
time64_t _time64(time64_t *t) { implementation }
#endif
time_t time(time_t *t) { implementation }
```
But latest VC complicates it by providing inline time implementation in time.h. As already established it is possible to mask inline implementation by defining _INC_TIME_INL, but problem with this approach is that nothing prevents them from changing it in next release, and then we would end up tracking multiple compiler versions. [Actually worse, multiple ucrt header sets.] And we don't want to do that. That's how second implementation ends up as
```
#if !defined(_WIN32) || !defined(_MSC_VER)
time_t time(time_t *t) { implementation }
#endif
```
This is *not* ideal, because some older _MSC_VER-sions can/will call `time` and above condition would then be in the way. However, even if it's not ideal, we can consider it good enough, because even if time is not overridden, test is still at least partially meaningful (right?). And not to forget that it is not ideal no matter what we do, because you can't override call from libssl.dll, not by providing implementation in .exe that is. So that this exercise is meaningless in *default* configuration anyway :-)